### PR TITLE
RWS entry for John Deere

### PR DESCRIPTION
**Pull request by ryber**

### CODE REVIEW

**Review by shuranhuang — APPROVED** [review]
LGTM

File: related_website_sets.JSON [hunk 1]
```diff
@@ -387,6 +387,16 @@
         ]
       }
     },
+    {
+      "contact": "bergmanpatrickr@johndeere.com",
+      "primary": "https://johndeere.com",
+
+      "associatedSites": ["https://deere.com"],
+
```
Comment (Parasos) [inline related_website_sets.JSON:395]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

File: related_website_sets.JSON [hunk 2]
```diff
@@ -387,6 +387,16 @@
         ]
       }
     },
+    {
+      "contact": "bergmanpatrickr@johndeere.com",
+      "primary": "https://johndeere.com",
```
Comment (Parasos) [inline related_website_sets.JSON:392]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Comment by Parasos** [comment]
related_website_sets.JSON

**Pull request merged**